src/poseidon/zero_items.rs

```diff
@@ -0,0 +1,1212 @@
+    ]),
+    BlsScalar([
+        0x289cb5794acd3792,
+        0xdeb26e5fd23a5902,
+        0xb54518fb306a3eb2,
+        0x301d685faffd5e5a,
+    ]),
+    BlsScalar([
+        0x5399ecde2feb14ce,
+        0x2ae8e091fef7f244,
+        0xa203a01a3714b1f0,
+        0x5428feec51810432,
+    ]),
+    BlsScalar([
+        0x49f9d153863ba461,
+        0x91ed411177d01158,
+        0xb6ea8fde3b7216e6,
+        0x12e33b9519eeb9bb,
+    ]),
+    BlsScalar([
+        0x95dbef791ece50ae,
+        0x670bdbfef7c1dd3d,
+        0x623c56dafa4be47b,
+        0x2f0e79e1b71412bc,
+    ]),
+    BlsScalar([
+        0x0b4c19e32bf6fc67,
+        0x6ce1e3166ba07f34,
+        0xdbf5d28d5c2ab6b8,
+        0x095bd97314ccbdcb,
+    ]),
+    BlsScalar([
+        0x15812790a8c3f73b,
+        0xf4250eb85913ce09,
+        0x8005f814d18b235f,
+        0x44728c591cca5318,
+    ]),
+    BlsScalar([
+        0xd126c1b2d7e9d71e,
+        0xdbfd2574516a98eb,
+        0x5a4438c3c8f13225,
+        0x0ad4f2f8f709a942,
+    ]),
+    BlsScalar([
+        0x34440e43f3572614,
+        0x0cebd09260807cff,
+        0x0efdc68482021b5f,
+        0x3c3e795a747fee43,
+    ]),
+    BlsScalar([
+        0x052e8ae7b3585a57,
+        0x5d3e1e7d57f0e88a,
+        0xb628b9318105f9df,
+        0x25e86c86d634971b,
+    ]),
+    BlsScalar([
+        0x32a034f2dd46e4e1,
+        0x915bb1990dbb75b8,
+        0x46f6cf6503ad164c,
+        0x4238446f67f3e3bd,
+    ]),
+    BlsScalar([
+        0x3ca451fc26a4d0f4,
+        0x668b030d89698c6f,
+        0xac98fc70c94f0b1f,
+        0x0d11b12f1dfbdb88,
+    ]),
+    BlsScalar([
+        0x6a0a19414c917e39,
+        0x4a0446550d955ce2,
+        0x08e5040aedc5b4d7,
+        0x0d77390f48a05a5a,
+    ]),
+    BlsScalar([
+        0xe6531c521dd02d88,
+        0x5c42d2b8671ec92d,
+        0x918c3841436b3214,
+        0x69aff5f14f0b9a0f,
+    ]),
+    BlsScalar([
+        0xb3e50bdaf637d8d0,
+        0x142e22d6338b99c8,
+        0x22ee4aef181bcc3a,
+        0x3c011d3799532d7e,
+    ]),
+    BlsScalar([
+        0x715bda04f8fae8c4,
+        0x1092977cbf054381,
+        0x46b5ed98cc9d3262,
+        0x5845c507d274d7c9,
+    ]),
+    BlsScalar([
+        0x9c4e90fd8573b26e,
+        0xd48f80e18bc392d2,
+        0xda85b7a79c289587,
+        0x3b45e63e6a66c85d,
+    ]),
+    BlsScalar([
+        0xdea471afb0264a3c,
+        0x5614d63ca8ea592c,
+        0xcf895bdd95f490d0,
+        0x693151d1aeb2ec67,
+    ]),
+    BlsScalar([
+        0x8d1e924974418781,
+        0x797b7a1edac1f57e,
+        0xe850715f62aafab6,
+        0x486efaa4a602a3e4,
+    ]),
+    BlsScalar([
+        0xf7fbba3f4ac91b84,
+        0x6462e4218948a13f,
+        0x24a451b717d6b197,
+        0x2e30d871d64d3014,
+    ]),
+    BlsScalar([
+        0xaf06c9b60f763102,
+        0x8f45c6f9981385ad,
+        0xee3caecb80929b6e,
+        0x31d966d636f52c89,
+    ]),
+    BlsScalar([
+        0xa55b06fdbb766501,
+        0xd3291127521b4953,
+        0x3d431f057c38e610,
+        0x0c2625114241e1e4,
+    ]),
```

Comment:
We should probably comment these elements anyway. I know we said it would be enough t put a comment in the function, but now that I see it I changed my mind 😅

Reply:
see issue #31